discover/index.html

```diff
@@ -82,12 +82,6 @@ <h3 class="text-left" id="services"><i class="far fa-fw fa-map"></i> Available S
     </ul>
   <li><i class="fas fa-fw fa-unlock"></i> <code>Hillshade</code> – 10-meter statewide combined hillshade and slope
 </ul>
-<h3 class="text-left"><i class="far fa-fw fa-thumbs-up"></i> Best Practices</h3>
```

Comment:
So did I understand this correctly? Just remove the Best Practices section altogether?

Reply:
I think those two links. That would leave a few items still. I think we can remove the whole statistics section though

Reply:
Hmm...So do I just add it back in from an earlier draft? (I don't know how to reject edits or go back a draft or something!)